### ISSUE TITLE
Dark Theme.

### ISSUE THREAD

**Issue by iFarhaan** [USBA/padOS#1]
The theme is awesome but it would be great to have dark/darkest variant of this theme.

**Comment by kamalayadi** [comment]
Are you working on a dark variant of this theme ?

**Comment by iFarhaan** [comment]
No i'm not working on it. But i wish if i could work on that. i'm a tester more than a creator. As of now, from windows to macOS, android to IOS everything has dark mode why not this gorgeous theme?
@USBA

**Comment by kamalayadi** [comment]
Can you tell me if you will worked on it. This is for me the best theme than i have never seen, in black, it will be perfect !

**Comment by iFarhaan** [comment]
Yes!!! I've managed to make it's black variant but not as i expected. Even though my modification of black variant is awesome. Check this out. [https://imgur.com/Ve0ARsm](url)

**Comment by kamalayadi** [comment]
Thanks for your work but I can't see clearly the result, do you have other images ?

**Comment by kamalayadi** [comment]
I still wait the dark variant because there isn't an other theme like yours

**Comment by iFarhaan** [comment]
Check it by yourself...... Download it from here.... remember link validity only for 30 days. 

> 

> https://ufile.io/4topl97d [N.B. you can change font color according to your taste i love to keep it grey....]

**Comment by kamalayadi** [comment]
thanks !
